sed/config/default.yaml

```diff
@@ -32,6 +40,18 @@ dataframe:
   # binning factor used for the adc coordinate (2^(adc_binning-1))
   adc_binning: 2
 
+  # Flash loader requires the daq parameter and offset
+  daq: fl1user3
+  ubid_offset: 5
+  # The h5 file paths and aliases are defined under channels
+  # PulseId channel(s) are necessary
+  channels:
+      pulseId:
+      axis: 1
+      format: per_electron
+      group_name: "/uncategorised/FLASH.EXP/HEXTOF.DAQ/DLD1/"
+      slice: 2
+
```

Comment:
I realize now we have two different "formats" to address the channels in the hdf5 files: 
For mpes, we have a list of h5 groupnames, and a list of aliases to assign. 
For flash you have this dictionary tree for every channel. We could in prinicple also adopt this for mpes, then it is consistent. I would, however, not provide any channels in the default config, because this cannot be generic for all loaders, I think. Otherwise we shoould split this off into seperate sections. Maybe needs discussion.

Reply:
I think a dictionary like structure is more pythonic and the alias just becomes the key of each groupname/path (value).

You are right that it won't be generic anymore so I will remove it.

Reply:
We should probably provide a general default config, and examples for each experiment, that can be used to set up a per-site configuration. The mechanism for this is now available in PR #111